crates/agglayer-storage/src/columns/balance_tree_per_network/mod.rs

```diff
@@ -0,0 +1,64 @@
+use bincode::Options;
+use serde::{Deserialize, Serialize};
+
+use super::{default_bincode_options, Codec, ColumnSchema, BALANCE_TREE_PER_NETWORK_CF};
+use crate::{error::Error, types::Hash};
+
+#[cfg(test)]
+mod tests;
+
+/// Column family for the balance tree per network.
+///
+/// | ------------ key ------------ |    | ------------- value -------------- |
+/// | NetworkId "root"              | => | hash(node.left) hash(node.right)   |
+/// | NetworkId hash(node)          | => | hash(node.left) hash(node.right)   |
+/// | NetworkId hash(node)          | => | hash(leaf)                         |
+/// | NetworkId "leaves" hash(leaf) | => | leaf bytes array                   |
```

Comment:
The balances in the balance tree are `U256`, so this map is not really required. We could just store the balances directly.

Reply:
I wasn't sure about the payload of the leaves. Going to switch this to `U256`